client/state/sites/products/conflicts.ts

```diff
@@ -0,0 +1,66 @@
+/**
+ * Internal dependencies
+ */
+import createSelector from 'lib/create-selector';
+import { FEATURE_SPAM_AKISMET_PLUS } from 'lib/plans/constants';
+import {
+	PRODUCT_JETPACK_ANTISPAM,
+	PRODUCT_JETPACK_ANTISPAM_MONTHLY,
+	JETPACK_ANTISPAM_PRODUCTS,
+} from 'lib/products-values/constants';
+import { hasFeature } from 'state/sites/plans/selectors';
+import isSiteWPCOM from 'state/selectors/is-site-wpcom';
+import { hasSiteProduct } from 'state/sites/selectors';
+
+/**
+ * Type dependencies
+ */
+import type { AppState } from 'types';
```

Comment:
Out of curiosity, is `AppState` what we should use to type the state of the app? This is the first time I see this type and I'm wondering if it's replacing `DefaultRootState`.

Reply:
No idea to be honest! I see either one used. The Calypso state as a whole is way too big to properly type, so `AppState` is an alias for `any`.